ProjectLighthouse/PlayerData/Score.cs

```diff
@@ -21,6 +21,13 @@ public class Score
     [ForeignKey(nameof(SlotId))]
     public Slot Slot { get; set; }
 
+    [XmlIgnore]
+    public int? AdvSlotId { get; set; }
```

Comment:
I recommend changing this to something else because the Adv abbreviation can be kind of confusing. I came up with `ChildSlotId` but I'm open to suggestions